core/src/version.rc

```diff
@@ -23,7 +23,7 @@
         VALUE "InternalName", "Keyman Core"
         VALUE "LegalCopyright", "© SIL International"
         VALUE "LegalTrademarks", ""
-        VALUE "OriginalFilename", "KMNCORE0-0.DLL"
+        VALUE "OriginalFilename", "KEYMANCORE1-0.DLL"
```

Comment:
Here we have KEYMANCORE1-0.DLL, but in kmdev.wxs and elsewhere we have keymancore1.dll.

```suggestion
        VALUE "OriginalFilename", "KEYMANCORE1.DLL"
```